lib/ansible/template/native_helpers.py

```diff
@@ -15,7 +15,7 @@
 from ansible.module_utils._text import to_text
 from ansible.module_utils.common.collections import is_sequence, Mapping
 from ansible.module_utils.common.text.converters import container_to_text
-from ansible.module_utils.six import PY2, text_type, string_types
+from ansible.module_utils.six import text_type, string_types
```

Comment:
Do we plan to move away from using `six` in the controller code as well (in a separate PR)?

Reply:
I hadn't really given it much thought, but you are correct, it probably doesn't make much sense to use going forward.

Reply:
String/bytes helpers still make sense, though. At least for as long as we don't have type checking.